### PR TITLE
Group unreleased release notes by topic

### PR DESCRIPTION
**Pull request by mfussenegger**
## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Review by mkleen — APPROVED** [review]

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -63,45 +87,38 @@ Changes
...
-  enterprise feature.
 
-- Added the ``to_char`` scalar function for ``timestamp`` and ``interval``
-  argument data types.
+Administration and Operations
+-----------------------------
 
-- Added support for the ``split_part`` scalar function
+- Added information about the shards located on the node to the :ref:`NodeInfo
+  MXBean <node_info_mxbean>` which is available as an enterprise feature.
```
Comment (amotl) [inline docs/appendices/release-notes/unreleased.rst:102]: Are :ref:`` links able to span a line break?

**Review by amotl — APPROVED** [review]
👍

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/unreleased.rst:102]: Yep - see the rendered version of the PR:  https://crate--10927.org.readthedocs.build/en/10927/appendices/release-notes/unreleased.html#administration-and-operations

**Comment by amotl** [comment]
```
/home/runner/work/crate/crate/docs/general/user-defined-functions.rst.rst:207: WARNING: broken link: https://www.ecma-international.org/ecma-262/10.0/ (404 Client Error: Not Found for url: https://www.ecma-international.org/ecma-262/10.0/)
```

Better link to the base url https://www.ecma-international.org/publications-and-standards/standards/ecma-262/

**Comment by mfussenegger** [comment]
> ```
> /home/runner/work/crate/crate/docs/general/user-defined-functions.rst.rst:207: WARNING: broken link: https://www.ecma-international.org/ecma-262/10.0/ (404 Client Error: Not Found for url: https://www.ecma-international.org/ecma-262/10.0/)
> ```
> 
> Better link to the base url https://www.ecma-international.org/publications-and-standards/standards/ecma-262/

I opened up another PR to fix the link - see https://github.com/crate/crate/pull/10929

**Pull request merged**